### PR TITLE
fix(renderer): no_next_padding to fix indentation set to 1

### PR DESCRIPTION
**Pull request by pysan3**
## TLDR

Indentation is broken when parent `indent_marker` is not shown in the node since it is the last element in the tree.

This happens only and only if value is exactly 1. See the indent of `./src/B/{a,b}.js`.

- `= 0`

image

- `= 1`

image

- `= 2` (`>= 2` all works)
image

## Expected

When the indent is set to 1, the indent should only show `indent_marker` (or `last_indent_marker`) so that the indent width will be 1.
- `= 1`
image

---

### Main Issue

Closes https://github.com/nvim-neo-tree/neo-tree.nvim/issues/924

### Detailed Explanation

https://github.com/nvim-neo-tree/neo-tree.nvim/pull/864#issuecomment-1578027693

### Fix

https://github.com/nvim-neo-tree/neo-tree.nvim/pull/864#issuecomment-1579093376

### CODE REVIEW

**Review by cseickel — APPROVED** [review]
Nice and simple, looks good!
